pulsar-broker/src/main/java/com/yahoo/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -150,9 +150,14 @@ private NamespaceBundle getFullBundle(NamespaceName fqnn) throws Exception {
         return bundleFactory.getFullBundle(fqnn);
     }
 
-    private static final Deserializer<ServiceLookupData> serviceLookupDataDeserializer = (key, content) ->
-            jsonMapper().readValue(content, ServiceLookupData.class);
-
+    private static final Deserializer<ServiceLookupData> serviceLookupDataDeserializer = (key, content) -> {
+        final String jsonString = new String(content);
+        if (jsonString.contains("\"allocatedCPU\":")) {
```

Comment:
Actually as we have addressed it in #338 and 
- Both `LoadReport` and `LocalBrokerData` extends `ServiceLookupData` which has same getter name for broker-urls
- [ObjectMapperFactory](https://github.com/yahoo/pulsar/blob/master/pulsar-common/src/main/java/com/yahoo/pulsar/common/util/ObjectMapperFactory.java) doesn't fail on unknown fields
So, any deserializer will always be able to parse both load-report json so, I think #338 will be enough. Do, you think we should have logic for falling back on top of #338 ?

@bobbeyreese can you rebase with master to get the change and if we think to fall back then we can add logic on top of it.